### ISSUE TITLE
Fix unreachable source maps

### ISSUE THREAD

**Issue by danielbeutner** [marvinhagemeister/color-helpers#11]
Hi there,

I hope you're well!

it seems the generated source maps are pointing to `src/` but the packages provides `dist/`.

Since I'am using CRA 5 with webpack 5, it throws these warnings:

```
WARNING in ./node_modules/color-helpers/dist/es/conversion.js
Module Warning (from ./node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/me/Repos/xxx/node_modules/color-helpers/src/conversion.ts' file: Error: ENOENT: no such file or directory, open '/Users/me/Repos/xxx/node_modules/color-helpers/src/conversion.ts'

WARNING in ./node_modules/color-helpers/dist/es/index.js
Module Warning (from ./node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/me/Repos/xxx/node_modules/color-helpers/src/index.ts' file: Error: ENOENT: no such file or directory, open '/Users/me/Repos/xxx/node_modules/color-helpers/src/index.ts'

WARNING in ./node_modules/color-helpers/dist/es/operation.js
Module Warning (from ./node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/me/Repos/xxx/node_modules/color-helpers/src/operation.ts' file: Error: ENOENT: no such file or directory, open '/Users/me/Repos/xxx/node_modules/color-helpers/src/operation.ts'
```

Sending some love from the former team,

Daniel B.

**Comment by marvinhagemeister** [comment]
Hi Daniel, thanks for reaching out. Hope you're well too.

Took a look and turns out that the source maps were missing the `sourcesContent` property which typically holds the original source code. Just published a new version `2.0.1` with the fix.

Cheers

**Comment by danielbeutner** [comment]
Hey @marvinhagemeister!

thank you so much. Didn't know about the sourcesContent property.

I will install the new version it right away.

Cheers, Daniel

**Comment by marvinhagemeister** [comment]
@danielbeutner  No need, it was an error in this library. And the fix is included in `2.0.1`. Just updating to that version should resolve it on your end 👍
